src/main/java/com/puppycrawl/tools/checkstyle/checks/imports/RedundantImportCheck.java

```diff
@@ -48,6 +48,16 @@
  * &lt;module name="RedundantImport"/&gt;
  * </pre>
  * <p>
+ * Example:
+ * </p>
+ * <pre>
+ * package Test; // OK
+ * import Test.test; // violation, Imported from the same package as the current package
+ * import java.lang.String; // violation, the class non-statically imported is from the package
```

Comment:
`the class non-statically imported is from the package` -> `the class non-statically imported is from the 'java.lang' package`

Reply:
If this message violates line length, I think you could drop `non-statically`, since this is clear from import statement itself.

Reply:
Resolved.